packages/modals/src/elements/DrawerModal/DrawerModal.tsx

```diff
@@ -49,12 +58,32 @@ const DrawerModalComponent = forwardRef<HTMLDivElement, IDrawerModalProps>(
       useModal({
         idPrefix: id,
         modalRef,
-        focusOnMount,
-        restoreFocus,
+        focusOnMount: false /* [1:b] */,
+        restoreFocus: false /* [1:b] */,
         environment,
         onClose
       });
 
+    /* [1:c] */
+    useEffect(() => {
+      if (environment) {
+        if (modalRef.current && isOpen) {
+          if (restoreFocus === undefined ? true : restoreFocus) {
+            triggerRef.current = activeElement(environment) as HTMLElement;
+          }
+
+          if (focusOnMount === undefined ? true : focusOnMount) {
```

Comment:
Both `restoreFocus` and `focusOnMount` revert to `true` when undefined to preserve the existing behavior that the `useFocusJail` + `useModal` containers had in place.

Reply:
Alternatively, we can use defaultProps in this case if we think thats best.

Reply:
Default props makes sense to me as long as this prop reference is consistent everywhere else too.

Reply:
Agreed, will update.